File: programmers/lv0/코딩기초트레이닝/연산/두수의연산값비교하기/Solution.cs
using System;

public class Solution {
    public int solution(int a, int b) {
        int x = int.Parse(a.ToString() + b);
        int y = 2 * a * b;
        if (x == y || x > y) return x;
        return y;

        // Simple
        int x = int.Parse($"{a}{b}");
        int y = 2 * a * b;
        return Math.Max(x, y);
    }
}

File: programmers/lv0/코딩기초트레이닝/연산/두수의연산값비교하기/Solution.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) {
    int x = stoi(to_string(a) + to_string(b));
    int y = 2 * a * b;
    if (x == y) {
        return x;
    }
    if (x > y) return x;
    return y;
}